### ISSUE TITLE
Different number of reads by barcode

### ISSUE THREAD

**Issue by valery-shap** [tleonardi/pycoQC#130]
Hello,

version of PycoQC 2.5.2. There are no errors, but the number of reads by barcode in html file is different from counting by other methods.
I tried to count pass reads by bash (zcat |wc -l) /4|bc
and I parsed the sequencing_summary file with conditions:
'barcode_arrangement' == 'barcode02' & 'passes_filtering' == True
The number is identical between counting by bash and parsing.
What could be the mistake?

Best regards,
Valery

**Comment by OKyne1** [comment]
Yep I encountered this problem too and spent a long time trying to find missing reads. Thanks for pointing it out valery-shap.
